test/integration/api/create_account.sh

```diff
@@ -20,6 +20,8 @@ while true
 do STATUS=$(oc get account ${OSD_STAGING_1_ACCOUNT_CR_NAME_OSD} -n ${NAMESPACE} -o json | jq -r '.status.state');
     if [ "$STATUS" == "Ready" ]; then
         break
+    elif [ "$STATUS" == "InitializingRegions" ]; then
```

Comment:
I had to add this, as otherwise the accountclaim would not be linked to the account.

So I have to create the claim first, then the account, which seems to immediately force the account into `InitializingRegions`.

Maybe someone knows a better way to do this.